### ISSUE TITLE
Bounding box issue with multiline text

### ISSUE THREAD

**Issue by triathematician** [triathematician/blaisemath#263]
**Describe the bug**

**To Reproduce**

Text with two line breaks `\n\n`
Sample exception:
```
Caused by: java.lang.NullPointerException
	at java.desktop/java.awt.geom.Rectangle2D.union(Rectangle2D.java:774)
	at java.desktop/java.awt.geom.Rectangle2D$Double.createUnion(Rectangle2D.java:474)
	at com.googlecode.blaisemath.graphics.swing.render.WrappedTextRenderer.boundingBox(WrappedTextRenderer.java:162)
	at com.googlecode.blaisemath.graphics.swing.render.WrappedTextRenderer.boundingBox(WrappedTextRenderer.java:55)
	at com.googlecode.blaisemath.graphics.PrimitiveGraphicSupport.boundingBox(PrimitiveGraphicSupport.java:117)
	at com.googlecode.blaisemath.graphics.GraphicUtils.lambda$boundingBox$0(GraphicUtils.java:140)
	at com.googlecode.blaisemath.graphics.GraphicUtils.boundingBox(GraphicUtils.java:154)
	at com.googlecode.blaisemath.graphics.GraphicUtils.boundingBox(GraphicUtils.java:140)
	at com.googlecode.blaisemath.graphics.GraphicComposite.boundingBox(GraphicComposite.java:297)
	at com.googlecode.blaisemath.graphics.swing.JGraphicComponent.zoomToAll(JGraphicComponent.java:280)
```

**Additional context**

As part of making a bounding box for the text, the code eventually reaches BlaiseGraphics' `WrappedTextRenderer#computeLineBreaks`. Within that method, a string is separated into a list of strings, accounting for both newline breaks and for keeping the text within the bounds of a given width and height, with the provided font.

When running through the string, when it encounters a newline, it (barring edge cases and box sizing that aren't important here):

1. Creates a substring starting from one character past the end of the last substring it made (or the start of the entire string if not done yet) ending at the character just before the newline.
1. Add the trimmed version of that substring to the list of strings to return
1. Sets the start of the next substring variable (`pos0`) to be one character before the current ending index (`pos1`). If `pos0`'s index points to a newline, move it forward one.
1. Move `pos1` forward one character so it starts creating a new string.

The issue is that this only checks one '\\n'! As a consequence, it's only prepared to handle the case of one newline character. In our error causing condition, there's two of them. This causes `pos0` to point to the second newline character in "\\n\\n". When checking characters, the string is indexed by `pos1-1`, which at this time, is equal to `pos0` (see step 4). This points to the aforementioned newline, thus causing it to begin capturing a substring of length 0. To clarify, the solution isn't to start checking at `pos1` instead of `pos1-1`, since that would cause "\\n\\n\\n" to similarly break as "\\n\\n" does now, as the captured '\\n' would be trimmed.

This leads to an empty string being passed to the bounding box calculator (in `TextRenderer#boundingBox()`), which reasonably returns `null` on an empty string. The attempt to use this null in `WrappedTextRenderer#boundingBox()` (line 162) fails due to the null, causing the `NullPointerException` to be thrown.

In summary: BlaiseGraphics' `WrappedTextRenderer#computeLineBreaks` doesn't account for multiple newline characters in a row, causing a blank string to have a bounding box calculated for. The bounding box is null, but is attempted to be used, thus causing the exception.

**Comment by triathematician** [comment]
Note: this occurs with
```
        PrimitiveGraphic<AnchoredText, Graphics2D> textGraphic = new PrimitiveGraphic<>();
```
It does not occur with `LabeledShapeGraphic`
